newton/examples/__init__.py

```
@@ -395,76 +388,28 @@ def init(parser=None):
-        pipeline = newton.examples.create_collision_pipeline(model, args)
-        contacts = model.collide(state, collision_pipeline=pipeline)
-
-        # Using explicit parameters
-        pipeline = newton.examples.create_collision_pipeline(
-            model,
-            collision_pipeline_type="unified",
-            broad_phase_mode="nxn"
-        )
+        CollisionPipeline instance.
     """
     import newton  # noqa: PLC0415
```

Ruff (0.14.14)

[warning] Unused `noqa` directive (non-enabled: `PLC0415`)

Remove unused `noqa` directive

(RUF100)